clang/test/CXX/drs/dr5xx.cpp

```diff
@@ -661,6 +680,10 @@ namespace dr553 {
 }
 
 // dr554: na
+
+// dr555: na
```

Comment:
It might be worth copying these tests.
I agree that the changes are non-behavioral but I'd like @AaronBallman to give a second opinion

Reply:
I'm not keen to copy tests for an issue that is officially recognized as duplicate.

Reply:
Ah, I confused this with CWG550, which is a duplicate.
I still would like to avoid copying tests, though.

Reply:
Avoiding copying tests is a good idea, that just means we need to come up with different tests. ;-)

CWG466 is solely about cv-qualification and pseudo destructors. CWG555 is more about lookup issues and was resolved by P1131R2. We should be able to devise a different set of tests for both, unless I'm missing something?

Reply:
I added a proper test for 555. Basically, I copied 466 test, and derived reference tests from it. In 466, I derived a test for non-pointer and non-reference scalar types.

> Finally, there is no place in the Standard that describes the lookup for pseudo-destructor calls of the form p->T::~T() and r.T::~T(), where p and r are a pointer and reference to scalar, respectively.

As we agreed offline, I'm testing just this sentence from 555, as that's the only actionable thing there for us.